airflow/.docker/entrypoint.sh

```diff
@@ -60,13 +60,11 @@ wait_for_redis() {
 }
 
 install_csv_generator() {
-    # hack: will be removed once `csv_generator` is hosted on `pypi`
-    pip install --user csv-generator/dist/csv_generator-0.0.1-py3-none-any.whl
+    pip install --user /opt/wheels/csv_generator-0.0.1-py3-none-any.whl
```

Comment:
should we maybe change that to `*.whl`? (and change the method name to _install_wheels_)